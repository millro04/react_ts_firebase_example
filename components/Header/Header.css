.header-wrapper {
    text-align: center;
  }
  
.header-text {
    font-size: 36px;
  }
  