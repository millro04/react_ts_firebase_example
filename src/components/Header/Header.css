.header-wrapper {
    text-align: center;
    padding-top: 25px;
  }
  
.header-text {
    font-size: 36px;
  }

  .header-description {
    padding-top: 10px;
  }