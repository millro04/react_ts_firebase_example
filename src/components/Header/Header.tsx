import React from 'react';
import './Header.css';

const Header = () => {
  return (
    <div className="header-wrapper">
      <header className="header-text">
          Reading Tracker
      </header>
    </div>
  );
}

export default Header;
