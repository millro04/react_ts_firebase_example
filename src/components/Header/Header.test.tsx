import React from 'react';
import Header from './Header';
import { shallow, ShallowWrapper } from 'enzyme';


test('Header component renders', () => {
  const component: ShallowWrapper = shallow(<Header />);
  expect(component).toMatchSnapshot();
});
