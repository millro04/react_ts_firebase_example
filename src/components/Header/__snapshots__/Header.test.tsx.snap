// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header component renders 1`] = `
<div
  className="header-wrapper"
>
  <header
    className="header-text"
  >
    Reading Tracker
  </header>
</div>
`;
